packages/core/src/external/carequality/dq/process-inbound-dq.ts

```diff
@@ -37,15 +38,21 @@ export async function processInboundDocumentQuery(
     const destinationKey = createUploadFilePath(cxId, patientId, `${CCD_SUFFIX}.xml`);
     const ccdExists = await s3Utils.fileExists(bucket, destinationKey);
     if (!ccdExists) {
-      log("No CCD found. Let's generate one.");
+      log("No CCD found. Let's trigger generating one.");
       const queryParams = {
         cxId,
         patientId,
       };
       const params = new URLSearchParams(queryParams).toString();
-      const endpointUrl = `${apiUrl}/internal/docs/ccd`;
-      const url = `${endpointUrl}?${params}`;
-      await executeWithNetworkRetries(async () => await api.post(url), { log });
+
+      executeWithNetworkRetries(async () => api.post(`${apiUrl}/internal/docs/ccd?${params}`), {
```

Comment:
intentional? why is this unawaited and duplicated here and below?

Reply:
There was a discussion regarding this [here](https://metriport.slack.com/archives/C04DMKE9DME/p1719270581196869?thread_ts=1718897333.131819&cid=C04DMKE9DME). 

At this step, we're triggering generation of the full CCD. And in the meantime, we're calling another endpoint to very quickly generate an empty one

Reply:
So this is completely intentional